### PR TITLE
[COM-590] BrandLogos block

### PR DESCRIPTION
**Pull request by guilean**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?**
https://impulsumvc.atlassian.net/browse/COM-590

### CODE REVIEW

File: packages/blocks/src/BrandLogos/BrandLogos.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+const defaultArgs = {
+  logos: Array(8).fill('https://avatars0.githubusercontent.com/u/67131017?s=200'),
+  backgroundColor: 'background-page',
+  textVariant: 'textpairing-header-2XL',
+};
+
+export const WithText = Template.bind({});
+WithText.args = {
+  ...defaultArgs,
```
Comment (tkanzakic) [inline packages/blocks/src/BrandLogos/BrandLogos.stories.tsx:21]: this case should use a text variant, not a text pairing one

File: packages/blocks/src/BrandLogos/BrandLogos.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+export const WithTextPairing = Template.bind({});
+WithTextPairing.args = {
+  ...defaultArgs,
+  title: 'Title',
+  description: 'Description',
+};
+
+export const Playground = Template.bind({});
+Playground.args = defaultArgs;
```
Comment (tkanzakic) [inline packages/blocks/src/BrandLogos/BrandLogos.stories.tsx:33]: no way to set the title and or description for this case

File: packages/blocks/src/BrandLogos/BrandLogos.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+export const BrandLogos: FC<BrandLogosProps> = ({
+  backgroundColor = 'background-page',
+  title,
+  description,
+  logos,
+  textVariant,
+}) => (
+  <Flex
+    data-testid="brand-logos"
```
Comment (tkanzakic) [inline packages/blocks/src/BrandLogos/BrandLogos.tsx:24]: do we really need this?, I don't like having _test code_ in production, and if we really do, we should follow the `cmpsr.component.child-component` nomenclature

File: packages/blocks/src/BrandLogos/BrandLogos.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    bg={backgroundColor}
+    flexDirection="column"
+    alignItems="center"
+    py={{ base: '3.5rem', md: '4rem', lg: '5.5rem', xl: '7rem' }}
+    px={{ base: '2rem' }}
```
Comment (tkanzakic) [inline packages/blocks/src/BrandLogos/BrandLogos.tsx:29]:
```suggestion
    px="2rem"
```

File: packages/blocks/src/BrandLogos/BrandLogos.test.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    const brandLogos = screen.getByTestId('brand-logos');
+    const textPairing = brandLogos.firstChild.childNodes;
+    expect(textPairing).toHaveLength(2);
+  });
+  test('should render text when title or description are provided', () => {
+    givenComponentRendered({ title: 'Title' });
+    const brandLogos = screen.getByTestId('brand-logos');
+    const text = brandLogos.firstChild.childNodes;
+    expect(text).toHaveLength(1);
+  });
```
Comment (tkanzakic) [inline packages/blocks/src/BrandLogos/BrandLogos.test.tsx:25]: this is only verifying the title case, missing an equivalent test for the description

**Review by tkanzakic — CHANGES_REQUESTED** [review]
The design for an odd number of items does not match figma

Screenshot 2022-06-02 at 14 46 20

File: packages/blocks/src/BrandLogos/BrandLogos.stories.tsx (hunk 1 above)
Comment (guilean) [reply packages/blocks/src/BrandLogos/BrandLogos.stories.tsx:21]: Fixed

File: packages/blocks/src/BrandLogos/BrandLogos.stories.tsx (hunk 2 above)
Comment (guilean) [reply packages/blocks/src/BrandLogos/BrandLogos.stories.tsx]: Fixed

File: packages/blocks/src/BrandLogos/BrandLogos.test.tsx (hunk 5 above)
Comment (guilean) [reply packages/blocks/src/BrandLogos/BrandLogos.test.tsx]: Added

**Review by tkanzakic — APPROVED** [review]

**Pull request merged**